axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/AggregateBasedAxonServerEventStorageEngine.java

```diff
@@ -204,13 +202,19 @@ private MessageStream<EventMessage<?>> eventsForCriterion(SourcingCondition cond
         return MessageStream.fromStream(
                 aggregateStream.asStream(),
                 this::convertToMessage,
-                event -> Context.with(LegacyResources.AGGREGATE_IDENTIFIER_KEY, event.getAggregateIdentifier())
-                                .withResource(LegacyResources.AGGREGATE_TYPE_KEY, event.getAggregateType())
-                                .withResource(LegacyResources.AGGREGATE_SEQUENCE_NUMBER_KEY,
-                                              event.getAggregateSequenceNumber())
-                                .withResource(ConsistencyMarker.RESOURCE_KEY,
-                                              new AggregateBasedConsistencyMarker(event.getAggregateIdentifier(),
-                                                                                  event.getAggregateSequenceNumber())));
+                AggregateBasedAxonServerEventStorageEngine::buildContext
+        );
+    }
+
+    private static Context buildContext(Event event) {
```

Comment:
```suggestion
    private static Context buildSourceContext(Event event) {
```
I would prefer this name, because context is not "general", for stream is different and build inside AxonServerMessageStream.

Reply:
But a context is a thing we attach to every message in the message stream :thinking: 
Not sure I'd like `buildSourceContext` entirely. 

What about we do `eventContext(Event)`?